node_modules/.yarn-integrity

```diff
@@ -1 +1 @@
-2e6bdee5b01093c3fb20daeaae838b7c24e86ee68a22f5a1670a583b61581bdf
```

Comment:
Try to add the whole node_modules dir to .gitignore. If it takes you longer than 5 mins, lemme know and I'll do it